### ISSUE TITLE
Google analytics field not saving complete content

### ISSUE THREAD

**Issue by patrickbrouwers** [AsgardCms/Platform#58]
The default google analytics script is too big to be saved in that setting:

```
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
            (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

        ga('create', 'XXXXXXXX', 'auto');
        ga('send', 'pageview');
```

Anything that can be done make it possible?

**Comment by nWidart** [comment]
Hm that's a problem indeed. I see the column type in the database is set to *varchar*.
To be safe it might be better to change this to *text*, so that settings that require more data, can be completely saved.

**Comment by nWidart** [comment]
An easy - temporary- fix could to just save the `ga` code, and use that inside the full code in your template.

**Comment by patrickbrouwers** [comment]
Jep, that's how I'm doing it right now

**Comment by nWidart** [comment]
Has been fixed in version [1.0.3](https://github.com/AsgardCms/Setting/releases/tag/1.0.3) of the Settings module.
